src/explore-education-statistics-frontend/src/app/sitemap.ts

```diff
@@ -0,0 +1,113 @@
+import getPublicationSlugs from '@frontend/services/sitemapService';
+import { MetadataRoute } from 'next';
+
+// Priority is a decimal bounded [0.0, 1.0] which indicates to Google which pages we most want users to find in search results.
+// The default priority is 0.5.
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const publicationPages = await getPublicationSlugs();
+
+  return [
+    {
+      url: process.env.PUBLIC_URL,
+      lastModified: new Date(),
+      changeFrequency: 'monthly',
+      priority: 1,
+    },
+    {
+      url: `${process.env.PUBLIC_URL}not-found`,
```

Comment:
I did talk it through with someone at some point, but can't remember who. I think we'd all agree it would be nice to remove it (I actually flirted with a mini func to remove the slash if one was found so it works either way, but thought it too noddy). One for a rainy day perhaps